packages/@aws-cdk/aws-appmesh/package.json

```diff
@@ -96,7 +97,9 @@
     "@aws-cdk/aws-iam": "0.0.0",
     "@aws-cdk/aws-servicediscovery": "0.0.0",
     "@aws-cdk/core": "0.0.0",
-    "constructs": "^3.2.0"
+    "constructs": "^3.2.0",
+    "@aws-cdk/aws-acmpca": "0.0.0",
```

Comment:
nit: Alphabetical order of the dependencies should be preserved. [Reference](https://github.com/aws/aws-cdk/pull/11563#discussion_r532961129)